### ISSUE TITLE
Fastly Magento 2 di:compile issue

### ISSUE THREAD

**Issue by battamich** [fastly/fastly-magento2#333]
The following error is shown by Magento (Cloud 2.3.2) when executing the di:compile command.
```
Errors during compilation:

Fastly\Cdn\Model\View\Asset\Image
Incompatible argument type: Required type: string. Actual type: \Fastly\Cdn\Model\View\Asset\filePath; File: /app/vendor/fastly/magento2/Model/View/Asset/Image.php

Total Errors Count: 1
```

**Comment by battamich** [comment]
The following old issue had the same problem: (Jun 19)
https://github.com/fastly/fastly-magento2/issues/285

**Comment by vvuksan** [comment]
This has been fixed in #334 and released as part of 1.2.124.
